gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -26,6 +26,116 @@ syntax = "proto2";
 option java_package = "com.google.transit.realtime";
 package transit_realtime;
 
+message StopTime{
+  required int64 stop_sequence = 1;
+  required uint64 arrival_time = 2;
+  required uint64 departure_time = 3;
+  required string stop_id = 4;
+  optional TranslatedString stop_headsign = 5;
+  optional PickupType pickup_type = 6;
+  optional DropOffType drop_off_type = 7;
+  optional float shape_dist_traveled = 8;
+
+  enum PickupType{
+    REGULAR_PICKUP = 0;
+    NO_PICKUP = 1;
+    MUST_PHONE_AGENCY = 2;
+    MUST_ASK_DRIVER = 3;
```

Comment:
When I try to compile this .proto to bindings using the [gtfs-realtime-bindings project](https://github.com/MobilityData/gtfs-realtime-bindings/blob/master/java/UPDATING.md) using `protoc --java_out=src/main/java --proto_path=.. ../gtfs-realtime.proto`, I get this error:

~~~
gtfs-realtime.proto:50:5: "MUST_ASK_DRIVER" is already defined in "transit_realtime.StopTime".
gtfs-realtime.proto:50:5: Note that enum values use C++ scoping rules, meaning that enum values are siblings of their type, not children of it.  There
fore, "MUST_ASK_DRIVER" must be unique within "transit_realtime.StopTime", not just within "DropOffType".
~~~

One possible fix is to rename this to `MUST_ASK_DRIVER_FOR_PICKUP`.